docs/source/host/start-hosting.rst

```diff
@@ -17,19 +13,19 @@ Overview of Hosting
 
 Vircadia's hosting interface consists of the following components that work together to create an immersive VR experience for your visitors:
 
-* The *Sandbox* is a testing environment hosted locally on your computer that gives you a space to play around and build content without affecting any domains in the metaverse. Note that it is possible to configure Vircadia so that your Sandbox is also a registered domain that is hosted directly from your local computer. The Sandbox application can be download from the `Vircadia website <https://www.highfidelity.com/download#sandbox>`_.
-* A *domain* is a registered location in the metaverse. Each domain is assigned a unique ID to identify it. You can sign up for a new domain in one of two ways: either by setting up a Vircadia server or by logging into your account via the Vircadia website.
-* A *server* hosts a domain, and processes incoming network requests for the domain. The server for your domain is either a physical computer or a cloud-hosted server.
-* A *droplet* or *cloud domain* is a cloud server hosted on DigitalOcean.
-* A *content set* is the content that you've designed to make your domain unique and pretty. 
+* The *Domain Server* is a virtual world that gives you a space to build content in.
+* A *domain* is a registered server in the metaverse. Each domain is assigned a unique ID to identify it. You can connect your domain to the metaverse by logging into your account via the Vircadia website.
```

Comment:
```suggestion
* A *domain* is a server in the metaverse. Each domain is assigned a unique ID to identify it. You can connect your domain to a Metaverse Server.
```

Reply:
I think that the specific reference to the Vircadia website will make the most sense until we have federation working and UX can easily make obvious how it all connects.

Reply:
```suggestion
* A *domain* is a server in the metaverse. Each domain is assigned a unique ID to identify it. You can connect your domain to the Vircadia Metaverse Server by logging into your account on the Vircadia website.
```
Yeah I guess so. How about this then?